hwdb.d/60-sensor.hwdb

```diff
@@ -962,6 +962,10 @@ sensor:modalias:acpi:INVN6500*:dmi:*:svnTOSHIBA:pnTOSHIBAENCORE2WT8-B:*
 # Toshiba Encore WT10A tablet
 sensor:modalias:acpi:INVN6500*:dmi:*:svnTOSHIBA:pnTOSHIBAWT10-A-103:*
  ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, 1
+ 
```

Comment:
Please drop space in this line.

Reply:
@yuwata 
I'm still finding my way around github, so thanks for your patience. I've removed the space on the branch in my forked repository which seems to have updated here and clicked `Add single comment` here. Do I need to do anything else, the help docs are not helping..

Reply:
the space is still present, you need to remove it, amend the commit and force push

Reply:
@bluca thanks. There were two spaces (lines 965 and 966) but I couldn't see the one on the empty line when I clicked `View reviewed changes`. I'm still a bit lost with the jargon though. Sorry for being a pain over 3 lines of code that will likely never be used by anyone else.